### PR TITLE
feat(ci): testing23

### PR DESCRIPTION
**Pull request by aali309**
# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
* [ ] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
* [ ] Linked a relevant issue which this PR resolves
* [ ] Linked any other relevant issues, PR's, or documentation, if any
* [ ] Resolved all conflicts, if any
* [ ] Rebased your branch PR on top of the latest upstream `main` branch
* [ ] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
* [ ] [Signed all commits using a GPG signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#gpg-commit-signature-verification)

**To recreate commits with GPG signature** `git fetch upstream && git rebase --force --gpg-sign upstream/main`
_______________________________________________

Fixes: #1

## Description of the change:
*This change allows an environment variable to be configured so that...*

## Motivation for the change:
*This change is helpful because users may want to...*

## How to manually test:
1. *Run CRYOSTAT_IMAGE=quay.io... sh smoketest.sh...*
2. *...*

### CODE REVIEW

File: .github/workflows/pr-request-review.yml [hunk 1]
```diff
@@ -30,4 +30,4 @@ jobs:
           fi
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
+#test23
```
Comment (mwangggg) [inline .github/workflows/pr-request-review.yml:33]: hellooo!!

**Review by mwangggg — COMMENTED** [review]
hello? uhhh

**Comment by aali309** [comment]
new

**Comment by aali309** [comment]
new comment

File: .github/workflows/pr-request-review.yml (hunk 1 above)
Comment (mwangggg) [inline .github/workflows/pr-request-review.yml:33]: review!

**Comment by aali309** [comment]
test

**Comment by aali309** [comment]
> /request_review edited for test

this is a test

File: .github/workflows/pr-request-review.yml (hunk 1 above)
Comment (ashihab20) [reply .github/workflows/pr-request-review.yml:33]: yes

**Review by ashihab20 — COMMENTED** [review]
test

**Comment by ashihab20** [comment]
testing2

**Review by ashihab20 — COMMENTED** [review]
confirm label removal test run

**Pull request closed without merging**